### ISSUE TITLE
JSON export

### ISSUE THREAD

**Issue by per-mathisen-arm** [SaschaWillems/opengles.gpuinfo.org#4]
Feature request: The JSON export option on vulkan.gpuinfo.org is super useful, and it would be really good to have the same type of functionality here.

**Comment by SaschaWillems** [comment]
Yeah, that would be nice addition. But sadly the OpenGL ES caps viewer uses XML and that XML isn't stored in the database either. So doing a json export like in the Vulkan database would be a good chunk of work for which I currently don't have the time for.
